aat/src/aat/resources/features/F-089/S-089.1.td.json

```diff
@@ -0,0 +1,824 @@
+{
+  "_guid_": "S-089.1",
+  "_extends_": "F-089_Test_Complex_Organisation",
+  "title": "must return 200 for case type details request",
+  "specs": [
+    "an active profile in CCD",
+    "contains a valid jurisdiction id",
+    "contains a valid caseType id",
+    "has the 200 OK code",
+    "contains case type details"
+  ],
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Content-Encoding": "gzip",
+      "Content-Type": "[[ANYTHING_PRESENT]]",
+      "Transfer-Encoding": "[[ANYTHING_PRESENT]]",
+      "Strict-Transport-Security": "[[ANY_STRING_NULLABLE]]"
+    },
+    "body": {
+      "id": "FT_ComplexOrganisation",
+      "description": "[[ANY_STRING_NOT_NULLABLE]]",
+      "version": {
+        "number": "[[ANY_INTEGER_NOT_NULLABLE]]",
+        "live_from": "[[ANY_STRING_NOT_NULLABLE]]",
+        "live_until": "[[ANY_STRING_NULLABLE]]"
+      },
+      "name": "FT-Complex Organisation",
+      "jurisdiction": {
+        "id": "BEFTA_MASTER",
+        "name": "BEFTA Master",
+        "description": "[[ANY_STRING_NOT_NULLABLE]]",
+        "live_from": "[[ANY_STRING_NOT_NULLABLE]]",
+        "live_until": "[[ANY_STRING_NULLABLE]]",
+        "case_types": [
+        ]
+      },
+      "events": [
+        {
+          "__operator__": "equivalent",
+          "__ordering__": "unordered",
+          "__elementId__": "id"
```

Comment:
`"__ordering__": "unordered",`
This line would be just enough. Others are so by default, already.